openshift/Makefile

```diff
@@ -16,5 +16,5 @@ test-e2e-downstream-stable:
 
 test-e2e-downstream:
 	@env RELEASE_YAML=$(RELEASE_YAML) ./e2e-tests-openshift.sh
-	@env LOCAL_CI_RUN=true TEST_YAML_IGNORES="kaniko s2i buildah jib-maven" ../test/e2e-tests.sh
```

Comment:
I think we need to skip these in downstream

Reply:
the whole point of this patch is to make them working downstream.... https://github.com/openshift/tektoncd-catalog/pull/57#issuecomment-555528899